diff --git a/src/defaults.ts b/src/defaults.ts
index 7e73c07..e456580 100644
--- a/src/defaults.ts
+++ b/src/defaults.ts
@@ -10,8 +10,8 @@ declare const __PLUS__: boolean;

 export const DEFAULT_COLORS = {
     darkScheme: {
-        background: '#181a1b',
-        text: '#e8e6e3',
+        background: '#@background@',
+        text: '#@text@',
     },
     lightScheme: {
         background: '#dcdad7',
@@ -20,7 +20,7 @@ export const DEFAULT_COLORS = {
 };

 export const DEFAULT_THEME: Theme = {
-    mode: 1,
+    mode: @darkMode@,
     brightness: 100,
     contrast: 100,
     grayscale: 0,
